portfolio/src/main/webapp/style.css

```diff
@@ -21,44 +24,37 @@ body {
 }
 
 h1 {
-  font-family: "Montserrat";
-  font-weight: 300;
-  font-size: 3em;
+  font-family: "Roboto";
+  font-size: 55;
   color: white;
+  margin-bottom: 45px;
+  margin-top: 60px;
 }
 
 h2 {
-  font-family: "Montserrat";
-  font-weight: 400;
-  font-size: 1.4em;
-  color: white;
+  font-family: "Roboto";
+  font-size: 15;
+  color: #414141;
   margin-top: 35px;
   letter-spacing: 0.5px;
 }
 
 p {
-  font-family: "Montserrat";
-  font-weight: 300;
-  font-size: 0.95em;
-  color: white;
+  font-family: "Roboto";
+  font-size: 10;
+  color: #414141;
   margin-bottom: 35px;
   letter-spacing: 0.5px;
 }
 
 li {
-  font-family: "Montserrat";
-  font-weight: 300;
-  font-size: 0.95em;
-  color: white;
+  font-family: "Roboto";
+  font-size: 10;
+  color: #414141;
   padding: 2.5px;
   letter-spacing: 0.5px;
```

Comment:
[Style](https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order): Alphabetize declarations.

Here and elsewhere.